test/reducers/ticketReducersTest.js

```diff
@@ -95,6 +105,17 @@ describe('Ticket Reducers', function test () {
       expect(Object.values(this.OnReceiveIncidentFromEmpty).filter(function (ticket) { return ticket.originId === 38808134 })[0]).to.exist
       expect(Object.values(this.OnReceiveIncidentFromPopulated).filter(function (ticket) { return ticket.originId === 38808134 })[0]).to.exist
     })
+
+    it('Should replace a given ticket with null values when REMOVE_TICKET is dispatched', function() {
+      const result = map(populatedTicketList, removeTicket(38502026))
+      expect(result[38502026]).to.be.null
```

Comment:
Another condition that would be helpful here is 

expect(result[44444444]).to.not.be.null 

to ensure that only the intended ticket is removed